### PR TITLE
Get min and max from filled histogram buckets

### PR DESCRIPTION
**Pull request by pirgeo**
- extract min and max from the histogram
- handle corner cases
- add tests

### CODE REVIEW

File: dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java [hunk 1]
```diff
@@ -192,4 +184,64 @@ static DimensionList fromLabels(Labels labels) {
...
+    for (int i = 0; i < pointData.getCounts().size(); i++) {
+      if (pointData.getCounts().get(i) > 0) {
+        // the current bucket contains something.
+        if (i == 0) {
+          // If we are in the first bucket, use the upper bound (which is the lowest specified bound
+          // overall) otherwise this would be -Inf, which is not allowed. This is not quite correct,
+          // but the best approximation we can get at this point. This might however lead to a min
+          // that is bigger than the sum, therefore we return the min of the sum and the lowest
+          // bound.
+          return Math.min(pointData.getBoundaries().get(i), pointData.getSum());
```
[the remark this answers is not in the document]
Comment (pirgeo) [reply dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java:210]: Probably. However, using a mean implies that the actual lowest value is smaller than the min boundary that we set here. When using sum, we might be off by a bigger margin, but we can ensure that the min we return is always smaller than the actual smallest value. This, however, might not be something that we have to enforce.

[the remark this answers is not in the document]
Comment (arminru) [reply dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java]: With your changes added in a87b4e1 it LGTM.

**Review by arminru — APPROVED** [review]
